vmdb/app/models/vmdb_database_lock.rb

```diff
@@ -0,0 +1,30 @@
+class VmdbDatabaseLock < ActiveRecord::Base
+  self.table_name = 'pg_locks'
```

Comment:
Tricky... I was looking for a migration to define the database, objid, objsubi, etc. columns and such...